test/mock-client/main.go

```diff
@@ -34,7 +34,6 @@ import (
 	"strings"
 	"time"
 
-	"github.com/dgrijalva/jwt-go"
```

Comment:
that's probably not intentional?

Reply:
Not sure what changed that, or why I `git add`ed it. Definitely not intentional.